### PR TITLE
Using the 'const' keyword to be consistent with ECMA6 features

### PR DESCRIPTION
**Pull request by kevjames3**
See Issue: https://github.com/dlmanning/gulp-sass/issues/624

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,8 @@ gulp.task('sass', function () {
 `gulp-sass` can be used in tandem with [gulp-sourcemaps](https://github.com/floridoo/gulp-sourcemaps) to generate source maps for the Sass to CSS compilation. You will need to initialize [gulp-sourcemaps](https://github.com/floridoo/gulp-sourcemaps) prior to running `gulp-sass` and write the source maps after.
 
 ```javascript
-var sourcemaps = require('gulp-sourcemaps');
+'use strict';
```
Comment (robatron) [inline README.md]: Not necessary for ES6 modules: https://stackoverflow.com/a/31685340

Comment (kevjames3) [reply README.md]: You need to do `use strict` for node 4 in order to enable ECMA6 features

**Comment by robatron** [comment]
I don't think so because `use strict` is related to ES5 in browser runtimes: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode

**Comment by xzyfer** [comment]
Thanks, but the documentation uses ES5 on purpose. See https://github.com/dlmanning/gulp-sass/issues/624#issuecomment-328804234

**Pull request closed without merging**